pkg/web_app/lib/src/account.dart

```diff
@@ -147,3 +161,35 @@ class _AccountNavWidget {
     }
   }
 }
+
+class _PkgAdminWidget {
+  Element _root;
+  void init(Element root) {
+    if (!pageData.isPackagePage) return;
+    _root = root;
+    _root.append(elem('h3', classes: ['title'], text: 'Self-admin'));
+    final discontinueToggle = elem(
+      'a',
```

Comment:
We need to use a button for actions of this nature. We might even want to do a simple javascript `confirm(....)` for this sort of thing.

Reply:
Agreed, but I'd do that in a subsequent PR, big enough already.